code/game/objects/items/stacks/packagewrap.dm

```diff
@@ -140,9 +140,8 @@
 
 /obj/item/delivery/attack_self(mob/user as mob)
 	user.drop_item(src, user.loc)
-	if(contents.len)
-		if(ishuman(user))
-			user.put_in_hands(contents[1])
+	for(var/obj/item/M in contents)
```

Comment:
This is purely a style point, but why did you use M? It has no functional bearing on the code but it seems a little confusing, since we normally use M for mob iteration.

Reply:
i'm quite new to this, just picked a random letter. what would i use for item iteration? I?

Reply:
Yeah that's the norm, O would also be fine for Object, or any name that just makes it obvious what the variable means, you could even use something long like content_item or droppable_item. As long as when you look at it in code you know what is being talked about.